Rubberduck.Resources/Inspections/InspectionNames.resx

```diff
@@ -446,4 +442,7 @@
   <data name="SuperfluousAnnotationArgumentInspection" xml:space="preserve">
     <value>Superfluous annotation arguments</value>
   </data>
+  <data name="MisleadingByRefParameterInspection" xml:space="preserve">
+    <value>ByRef is a misleading modifier because the parameter will be passed ByVal.</value>
```

Comment:
This might be a bit long for the label shown for the inspection.

Reply:
I worked over the resource strings for this inspection pretty diligently to get the presentation to look right.  Below is an example output, and my `Code Inspection` window is reasonably sized.  I think it presents fairly well:

![InspectionView](https://user-images.githubusercontent.com/16185628/96194675-c5a05300-0eff-11eb-9dcb-1a937b12dc44.png)

Reply:
I think I understand the comment a little better.  Even though the text may 'fit', a full sentence is a different kind of content than the other `InspectionName` resources.  So...perhaps the following modifications would be more consistent:

**InspectionName:** _Misleading ByRef parameter modifier_ 

**InspectionInfo:** _The last parameter (the 'Value' parameter) of property mutators are always passed ByVal.  This is true regardless of the presence or absence of a ByRef or ByVal modifier.  Exception: A UserDefinedType must always be passed ByRef even when it is the last parameter of a property mutator._